update-dependabot.ps1

```diff
@@ -0,0 +1,24 @@
+$outputPath = "./.dependabot/config.yml"
+
+Remove-Item $outputPath
+
+Add-Content $outputPath "version: 1"
```

Comment:
An alternative would be to build up a string, and then replacing the contents at once. This will open and close the file for each call to `Add-Content`. Note that I don't really mind this a lot, as the script is not run often.

Reply:
You're totally right, I went ahead and fixed that. The script runs in only a few seconds so it's pretty quick, but avoiding needless work is always a good idea.